src/components/LinkToRecord/LinkToRecord.astro

```diff
@@ -12,6 +13,11 @@ export type Props = {
 const { record, openInNewTab, ...props } = Astro.props;
 const locale = getLocale() as SiteLocale;
```

Comment:
Hmm I should've moved the `as SiteLocale` to the `getLocale()` function itself when I had the chance

Reply:
@Jamerrone is about to file an issue in setting locale based on server/client context. I think that could be a good issue to address this in as well